ddp-workspace/projects/ddp-sdk/src/lib/components/user/activities/userActivities.component.ts

```diff
@@ -64,7 +77,12 @@ import { tap, mergeMap } from 'rxjs/operators';
             <span class="dashboard-mobile-label" [innerHTML]="'SDK.UserActivities.ActivityStatus' | translate"></span>
             <div class="dashboard-status-container">
               <img class="dashboard-status-container__img" [attr.src]="domSanitizationService.bypassSecurityTrustUrl('data:image/svg+xml;base64,' + element.icon)">
+              <ng-container *ngIf="showQuestionCount(element); else showStatusCode">
+                {{ 'SDK.UserActivities.ActivityQuestionCount' | translate: { 'questionsAnswered': element.numQuestionsAnswered, 'questionTotal': element.numQuestions } }}
+              </ng-container>
+              <ng-template #showStatusCode>
               {{ getState(element.statusCode) }}
+              </ng-template>
             </div>
```

Comment:
This is the main change: we show either the state name or a count of questions answered.